bears/general/InvalidLinkBear.py

```diff
@@ -10,22 +10,23 @@
 class InvalidLinkBear(LocalBear):
 
     @classmethod
-    def check_prerequisites(cls):
-        code = InvalidLinkBear.get_status_code_or_error("http://www.google.com")
+    def check_prerequisites(cls, timeout=2):
```

Comment:
you can go ahead and remove the timout variable and use 2 diectly, since `check_prerequisites` gets called only on initialization, and doesn't get the timeout setting AFAIK